docs/documentation/ui_customization/topics/admin-console.adoc

```diff
@@ -0,0 +1,4 @@
+[[_admin_console]]
+== Customizing the Administration Console
```

Comment:
```suggestion
== Customizing the Admin Console
```

Reply:
In general we are calling it the Admin Console.  Maybe we should change it on the Welcome page.